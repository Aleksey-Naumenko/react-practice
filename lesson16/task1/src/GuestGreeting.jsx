import React from "react";

function GuestGreeting() {
    return <h1>Hi stranger âœŒ Please login ğŸš€</h1>;
}

export default GuestGreeting;