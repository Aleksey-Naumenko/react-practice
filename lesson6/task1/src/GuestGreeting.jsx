import React from 'react';

const GuestGreeting = () => {
    return <h1>Hi stranger ✌️ Please login 🚀</h1>
};

export default GuestGreeting;