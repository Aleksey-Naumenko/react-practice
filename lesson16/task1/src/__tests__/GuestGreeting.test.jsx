import React from "react";
import GuestGreeting from "../GuestGreeting";
import { shallow } from "enzyme";

describe("<GuestGreeting />", () => {
  it("should render correctly guest greeting", () => {
    const wrappedComponent = shallow(<GuestGreeting />);

    expect(wrappedComponent).toMatchSnapshot();
  });
});