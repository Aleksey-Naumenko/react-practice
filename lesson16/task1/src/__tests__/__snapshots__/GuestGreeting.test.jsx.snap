// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<GuestGreeting /> should render correctly guest greeting 1`] = `
<h1>
  Hi stranger âœŒ Please login ğŸš€
</h1>
`;
